src/lucky/routeable.cr

```diff
@@ -27,26 +40,71 @@ module Lucky::Routeable
     end
   end
 
+  # Define a nested route that responds to the appropriate HTTP request
+  # automatically
+  #
+  # This works similarly to `action` but it will provide multiple parameters.
+  # For example:
+  # ```
+  # class Posts::Comments::Show
+  #   nested_action do
```

Comment:
I think it would be good to also show what path this matches: `/posts/:post_id/comments/:id`